java-checks/src/test/java/org/sonar/java/checks/serialization/SerializableFieldInSerializableClassCheckTest.java

```diff
@@ -40,16 +41,16 @@ void test() {
   }
 
   @Test
-  void testWithoutSemantic() {
+  void test_without_semantic() {
     JavaCheckVerifier.newVerifier()
       .onFile(testSourcesPath("checks/serialization/SerializableFieldInSerializableClassCheck.java"))
       .withCheck(new SerializableFieldInSerializableClassCheck())
-      .withoutSemantic()
+      .withClassPath(Collections.emptyList())
```

Comment:
I'm not sure this is needed. `.withoutSemantic()` is supposed to do exactly this. To me, the test is already valid and the rule operating as expected.